packages/shared/src/components/RankProgressWrapper.tsx

```diff
@@ -34,11 +32,7 @@ export default function RankProgressWrapper({
   const { user } = useContext(AuthContext);
   const [showAccountDetails, setShowAccountDetails] = useState(false);
   const [showRanksModal, setShowRanksModal] = useState(false);
-  const { flags } = useContext(FeaturesContext);
-  const devCardLimit = parseInt(
-    getFeatureValue(Features.DevcardLimit, flags),
-    10,
-  );
```

Comment:
Nimrod mentioned that he would like to keep the support if this is already implemented.
One thing, I'd change is to make it to 50 by default instead of null.
I mean we will always limit the DevCard generation but the exact threshold is received from Flagsmith with a default of 50